packages/client/components/ScopePhaseAreaAddJira.tsx

```diff
@@ -50,14 +50,14 @@ const ScopePhaseAreaAddJira = (props: Props) => {
     meetingRef
   )
   const {teamId} = meeting
-  const importStories = () => {
+  const authJira = () => {
     AtlassianClientManager.openOAuth(atmosphere, teamId, mutationProps)
   }
   return (
     <AddJiraArea>
-      <AddJiraButton onClick={importStories} size={'medium'}>
+      <AddJiraButton onClick={authJira} size={'medium'}>
         <JiraSVG />
-        Import stories from Jira
+        Import issues from Jira
```

Comment:
+1 traditionally the team votes on stories, bugs are not estimated because by their very nature these are uncertain

Reply:
Once we integrate with Jira, we refer to everything as issues instead of stories, e.g. "Search issues", "New Issue". I think we should be consistent with one or the other. I'd be happy to update everything to "stories" if that's more accurate in Jira land

![Screenshot 2022-01-27 at 18 09 59](https://user-images.githubusercontent.com/39854876/151418474-135b90b9-e3fd-474d-8b99-9630f878bf7a.png)